orchagent/dash/dashhaorch.cpp

```diff
@@ -25,6 +57,125 @@ DashHaOrch::DashHaOrch(DBConnector *db, const vector<string> &tables, DashOrch *
 
     dash_ha_set_result_table_ = make_unique<Table>(app_state_db, APP_DASH_HA_SET_TABLE_NAME);
     dash_ha_scope_result_table_ = make_unique<Table>(app_state_db, APP_DASH_HA_SCOPE_TABLE_NAME);
+
+    m_dpuStateDbConnector = make_unique<DBConnector>("DPU_STATE_DB", 0);
```

Comment:
VS tests are failing, looks like we will need swsscommon changes: https://github.com/sonic-net/sonic-swss-common/blob/master/common/database_config.json

Reply:
on it now.